docs/pages/data_visualization_colors.js

```diff
@@ -0,0 +1,145 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, ColorSchemeProvider, Flex, Heading, Text } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import ColorTile from '../components/ColorTile.js';
+import Page from '../components/Page.js';
+import { capitalizeFirstLetter } from '../components/utils.js';
+
+const MAIN_STEPS = ['01', '02', '03', '04', '05', '06'];
+const EXTENDED_STEPS = ['07', '08', '09', '10', '11', '12'];
+
+type ColorCardProps = {|
+  colorScheme: 'light' | 'dark',
+|};
+function SemanticThemeExample({ colorScheme }: ColorCardProps): Node {
+  return (
+    <ColorSchemeProvider colorScheme={colorScheme} id={colorScheme}>
+      <Box color="default" padding={4}>
+        <Flex direction="column" gap={4}>
+          <Heading size="400">{capitalizeFirstLetter(colorScheme)} mode</Heading>
+          <Flex direction="column">
+            <ColorTile
+              textColor={colorScheme}
+              description="Success (Graph)"
+              fullTokenName="color-data-visualization-success-graph"
+            />
+            <ColorTile
+              textColor={colorScheme}
+              description="Success (Text/Icon)"
+              fullTokenName="color-data-visualization-success-text"
+            />
+          </Flex>
+          <ColorTile
+            textColor={colorScheme}
+            description="Error (Graph and Text)"
+            fullTokenName="color-data-visualization-error"
+          />
+        </Flex>
+        <Flex />
+      </Box>
+    </ColorSchemeProvider>
+  );
+}
+
+export default function ColorPage(): Node {
+  return (
+    <Page title="Color palette">
+      <PageHeader
+        name="Color palette"
+        description={`The data visualization color palette is used to represent discrete categories of data. This palette utilizes Gestalt color palette and is optimized for improving accessibility in data visualizations. The palette is comprised of a 12-color categorical palette and semantic colors.  `}
+        showSourceLink={false}
+      />
+      <MainSection
+        name="12-Color categorical palette"
+        description={`
+        This palette helps create a consistent brand image among visualizations across Pinterest products, while ensuring enough color contrast against background and between individual colors. There are 12 colors in the palette, and is divided into a main palette and an extended palette.
+
+        - The colors in the main palette are used in most cases.
+        - Use the extended palette only when more than 6 colors are needed.
+        - When there is only one color needed in a visualization, always use Data Visualization #05, which is reserved for showing total value.
```

Comment:
though not my personal preference, our [style guide](https://paper.dropbox.com/doc/Gestalt-Docs-Style-Guide--BiLzreOIFxwXuFHbkAGs4LD4Ag-aNRXaF9c45wzag4zQpS3l#:h2=Use-numeric-symbols-for-all-nu) indicates that we should use numbers for "1" here and below (I'm just concerned with the consistency as compared to the preceding "6")